helm/templates/gateway/gateway-ingress.yaml

```diff
@@ -1,10 +1,68 @@
-{{- if .Values.gateway.enabled -}}
-{{- if .Values.gateway.ingress.enabled -}}
+{{- if and .Values.gateway.enabled (not .Values.gateway.ingressController.enabled) -}}
 {{- $serviceGWName := include "gravitee.gateway.fullname" . -}}
+{{- $graviteeName := include "gravitee.name" . -}}
 {{- $serviceGWPort := .Values.gateway.service.externalPort -}}
 {{- $ingressPath   := .Values.gateway.ingress.path -}}
 {{- $ingressPathType   := .Values.gateway.ingress.pathType -}}
 {{- $apiVersion := include "common.capabilities.ingress.apiVersion" . -}}
+{{- $supportsIngressClassname := include "common.ingress.supportsIngressClassname" . -}}
+{{- if .Values.gateway.servers }}
+{{- range $i, $server := .Values.gateway.servers }}
```

Comment:
should we add a `{{- if server.ingress.enabled }}` condition here ? I can see that in the value file, we have this flag, but cannot find where it is used.